### ISSUE TITLE
Share feature should include raw coordinates

### ISSUE THREAD

**Issue by pigsonthewing** [organicmaps/organicmaps#5829]
When the user shares a location, for example by email, the shared text should include the raw WGS84 coordinates, and/or a GeoURI, as well as the  organicmaps.app URI as currently included

**Comment by patepelo** [comment]
I agree the share location could be restrucutered. For example, if I tap a location where I'm not, it still includes in the text "I'm here". And the link top open the location should be universal to be opened with any supporting app.

**Comment by biodranik** [comment]
See #2957 #4930 and likely other related issues, and also unfinished #1821
